apache-couchbase-mixin/README.md

```diff
@@ -0,0 +1,95 @@
+
+![First screenshot of the Apache Couchbase cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/apache-Couchbase/screenshots/Couchbase_overview_1.png)
+
+## Apache Couchbase Node Overview
+
+The Apache Couchbase node overview dashboard provides details on memory usage, cpu usage, network request info (request methods and response codes), query service request info (volume, type, and processing time), index service performance (request volume, cache hit ratio, and scan latency), and both error and couchdb logs. The default Couchbase system log path is `/opt/couchbase/var/lib/couchbase/logs` on Linux, or `C:\Program\Files\Couchbase\Server\var\lib\couchbase\logs` on Windows.
+
+![First screenshot of the Apache Couchbase node overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/apache-Couchbase/screenshots/Couchbase_nodes_1.png)
+
+Couchbase system logs are enabled by default in the `config.libsonnet` and can be removed by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+In order for the selectors to properly work for system logs ingested into your logs datasource, please also include the matching `instance`, `job`, and `cluster` labels onto the [scrape_configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics.
+
+```yaml
+scrape_configs:
+  - job_name: integrations/couchbase
+    static_configs:
+      - targets: [localhost]
+        labels:
+          job: integrations/couchbase
+          instance: "<your-instance-name>"
+          cluster: "<your-cluster-name>"
+          __path__: /opt/couchbase/var/lib/couchbase/logs/*.log
+```
+
+## Apache Couchbase Bucket Overview
+
+The Apache Bucket overview dashboard provides details on the top buckets based on key resource usage, items, operations, operations failed, high priority requests, cache hit ratio, number of replica vBuckets, and vBucket queue memory usage.
+
+![First screenshot of the Apache Couchbase bucket overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/apache-Couchbase/screenshots/Couchbase_overview_1.png)
+
+## Alerts Overview
+
+- ApacheCouchbaseHighCPUUsage: There is high cpu usage for a node.
+- ApacheCouchbaseHighMemoryUsage: There is a limited amount of memory available for a node.
+- ApacheCouchbaseMemoryEvictionRate: There is a spike in evictions in a bucket, which indicates high memory pressure.
+- ApacheCouchbaseInvalidRequestVolume: There is a high volume of incoming invalid requests, which may indicate a DOS or injection attack.
+
```

Comment:
We usually put a short section with the alert thresholds under the Alerts Overview, here is an example from SAP HANA:


Default thresholds can be configured in `config.libsonnet`.

```js
{
  _config+:: {
    alertsCriticalHighCpuUsage: 80, // percent 0-100
    alertsCriticalHighPhysicalMemoryUsage: 80, // percent 0-100
    alertsWarningLowMemAllocLimit: 90, // percent 0-100
    alertsCriticalHighMemoryUsage: 80, // percent 0-100
    alertsCriticalHighDiskUtilization: 80, //percent 0-100
    alertsCriticalHighSqlExecutionTime: 1, // second
    alertsCriticalHighReplicationShippingTime: 1, //second
  },
}
```